README.md

```diff
@@ -725,7 +725,7 @@ Inspired by [awesome-php](https://github.com/ziadoz/awesome-php).
 
 ## Interactive Interpreter
 
-*Interactive Python interpreters (REPL).*
+*Interactive Python interpreters (REPL). See: [awesome-jupyter](https://github.com/markusschanta/awesome-jupyter).*
```

Comment:
It would be great if you put this link under the `Jupyter Notebook (IPython)` entry.

Reply:
Done, moved the link.